City-Bureau/city-scrapers-wichita
# 🏗️ Build spider: WAMPO – Transportation Policy Body

## What's this PR do?
Adds a spider to scrape meetings from the website of Wichita Area Metropolitan Planning Organization - Transportation Policy Body. The new spider is called `wicks_wampo_tpb`.

## Why are we doing this?
Requested by our site partners.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl wicks_wampo_tpb -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here](https://www.wampo.org/transportation-policy-body).

## Are there any smells or added technical debt to note?
- Page is lacking some key details but has great links for attachments. Certain details are hardcoded.
